src/wp-includes/deprecated.php

```diff
@@ -5870,3 +5870,85 @@ function _wp_theme_json_webfonts_handler() {
 	add_action( 'wp_enqueue_scripts', $fn_generate_and_enqueue_styles );
 	add_action( 'admin_init', $fn_generate_and_enqueue_editor_styles );
 }
+
+/**
+ * Prints the CSS in the embed iframe header.
+ *
+ * @since 4.4.0
```

Comment:
The newly deprecated functions in this file should all receive `@deprecated` annotations.